frontend/client/src/screens/CodeValidations.js

```diff
@@ -199,12 +213,19 @@ const CodeValidationsBase = observer((props) => {
               min={getFourteenDaysAgoString()}
               max={getTodayString()}
               onChange={handleDate}
+              placeholder="mm-dd-yyyy"
             ></input>
           </form>
           <p className="date-desc">Time zone set to {getDefaultTimezoneString()}</p>
         </div>
       </div>
 
+      {/* does this render a calendar picker upon clicking?  or a bunch of numbers vertically? */}
```

Comment:
I want to see if this is occurring on other machines/browser versions.  Our React version 16.x doesn't seem to conflict w the [ReactDatePicker docs](https://reactdatepicker.com/).

But a strange behavior is occurring that I couldn't figure out exactly why.  See below.  Is this happening to you all also?  Note this is happening on all browsers I tried: Chrome, Safari, Brave.

<img width="423" alt="Screen Shot 2020-08-13 at 4 32 20 PM" src="https://user-images.githubusercontent.com/16062590/90197125-b7b54100-dd9b-11ea-84b4-c2791670e92c.png">